homeassistant/components/marytts/tts.py

```diff
@@ -49,57 +50,43 @@ class MaryTTSProvider(Provider):
     def __init__(self, hass, conf):
         """Init MaryTTS TTS service."""
         self.hass = hass
-        self._host = conf.get(CONF_HOST)
-        self._port = conf.get(CONF_PORT)
-        self._codec = conf.get(CONF_CODEC)
-        self._voice = conf.get(CONF_VOICE)
-        self._language = conf.get(CONF_LANG)
+        self._mary = MaryTTS(
+            conf.get(CONF_HOST),
+            conf.get(CONF_PORT),
+            conf.get(CONF_CODEC),
+            conf.get(CONF_LANG),
+            conf.get(CONF_VOICE),
+        )
+        self._effects = conf.get(CONF_EFFECT)
         self.name = "MaryTTS"
 
     @property
     def default_language(self):
         """Return the default language."""
-        return self._language
+        return self._mary.locale
 
     @property
     def supported_languages(self):
         """Return list of supported languages."""
         return SUPPORT_LANGUAGES
 
-    async def async_get_tts_audio(self, message, language, options=None):
-        """Load TTS from MaryTTS."""
-        websession = async_get_clientsession(self.hass)
-
-        actual_language = re.sub("-", "_", language)
-
-        try:
-            with async_timeout.timeout(10):
-                url = f"http://{self._host}:{self._port}/process?"
-
-                audio = self._codec.upper()
-                if audio == "WAV":
-                    audio = "WAVE"
+    @property
+    def default_options(self):
+        """Return dict include default options."""
+        return {CONF_EFFECT: DEFAULT_EFFECTS}
```

Comment:
Why is that static? The user can overwrite the defaults per initial config

Reply:
Jup, you are correct. With the latest commit, the defaults are the initial effect settings